### DISCUSSION TITLE
10k手写公式数据集，希望可以帮助你改进！

### DISCUSSION THREAD

**Discussion by UUTTR9** [OleehyO/TexTeller#45] [Show and tell]
分享文件：train_dataset(1).zip
链接：https://pan.xunlei.com/s/VNunHL_PrRCgxrdFretqBDcMA1#
提取码：bw55
复制这段内容后打开手机迅雷App，查看更方便

**Comment by OleehyO** [comment]
刚好打算解决手写公式的问题，非常感谢！

**Comment by pjpanadas** [comment]
> 分享文件：train_dataset(1).zip 链接：https://pan.xunlei.com/s/VNunHL_PrRCgxrdFretqBDcMA1# 提取码：bw55 复制这段内容后打开手机迅雷App，查看更方便

链接失效了，能否再提供下呢，非常感谢

**Comment by OleehyO** [comment]
> 分享文件：train_dataset(1).zip 链接：https://pan.xunlei.com/s/VNunHL_PrRCgxrdFretqBDcMA1# 提取码：bw55 复制这段内容后打开手机迅雷App，查看更方便

Hello, 想问一下这个数据集有引用的来源吗？

**Comment by weichong1994** [comment]
> 分享文件：train_dataset(1).zip 链接：https://pan.xunlei.com/s/VNunHL_PrRCgxrdFretqBDcMA1# 提取码：bw55 复制这段内容后打开手机迅雷App，查看更方便

Hi，链接失效了可以提供一个新的链接嘛？非常感谢！
